### ISSUE TITLE
processed data download - folder naming

### ISSUE THREAD

**Issue by StephMonaco** [ispyb/EXI#265]
Feedback from User:
- if downloaded from the session summary at a sample level for a particular pipeline; Could the folder name also contain the protein acronym-sample name and run number?
![capture](https://user-images.githubusercontent.com/25433590/27441328-79c1dfdc-576d-11e7-8401-1270e6f63636.PNG)
![capture1](https://user-images.githubusercontent.com/25433590/27441326-79c01558-576d-11e7-8b84-6140f32b2a30.PNG)
![capture2](https://user-images.githubusercontent.com/25433590/27441327-79c11aca-576d-11e7-9fdd-75190228cfc7.PNG)
and instead of XDSAPP_DataprocID.zip having ACRO-SAMPLE NAME-XDSAPP_DataprocID.zip?

- if downloaded from the View all data collect interface for a sample : downloads all autoproc for that sample with the folder name : **download.zip** which makes it impossible for a user to download several autoproc that way for various samples.
![capture3](https://user-images.githubusercontent.com/25433590/27441676-909b3cc0-576e-11e7-8ae9-6153592d6028.PNG)
![capture5](https://user-images.githubusercontent.com/25433590/27441678-90a10cb8-576e-11e7-8175-575e3c4dc9f9.PNG)
![capture6](https://user-images.githubusercontent.com/25433590/27441677-909dbffe-576e-11e7-9cbf-c65198b5780a.PNG)
instead of **download.zip** having **ACRO-SAMPLE NAME-datacollectionID.zip**? or at least datacollectionID.zip?

**Comment by StephMonaco** [comment]
feedback from a user:
they prefer image prefix (up to run #) : ACRO-SampleName_1_AutoProcFilesFromDC for the whole zip
ACRO-SampleName_1_PIPELINENAME for the zip corresponding to the best result (in view all collections)

**Comment by antolinos** [comment]
Hard to parse this for me.

I propose for a single result:
```
var fileName = data[i].DataCollection_imagePrefix;                         
if (data[i].v_datacollection_summary_phasing_anomalous){
        fileName = fileName +  "_anomalous";
}
if (data[i].v_datacollection_summary_phasing_autoproc_space_group){
        fileName = fileName +  "_" + data[i].v_datacollection_summary_phasing_autoproc_space_group.replace(/\s+/g, '');
}
fileName = fileName + "_" +  data[i].v_datacollection_processingPrograms + ".zip"; 
```

Example: **PREFIX_anomalous__P4212_grenades_parallelproc.zip**


For the whole data collection:
**PREFIX_online_analysis.zip**

**Comment by antolinos** [comment]
@StephMonaco 
Is this done, right?

**Comment by StephMonaco** [comment]
Yes! thanks
